ProjFS.Mac/PrjFSKext/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -160,6 +172,10 @@ VirtualizationRootHandle VirtualizationRoot_FindForVnode(
         }
         
         vnode_t parent = vnode_getparent(vnode);
+        if (nullptr == parent)
+        {
+            KextLog_FileNote(vnode, "VirtualizationRoot_FindForVnode: vnode_getparent returned nullptr on vnode that is not root of a mount point");
+        }
         vnode_put(vnode);
         vnode = parent;
```

Comment:
Could this create problems further on if parent was null?

Reply:
Yes; dealing with the fallout from this is something that still needs to be done and is out of scope of this PR. See [issue #338](https://github.com/Microsoft/VFSForGit/issues/338).